cpu/cpu_windows.go

```diff
@@ -63,15 +62,17 @@ func CPUInfo() ([]CPUInfoStat, error) {
 	if err != nil {
 		return ret, err
 	}
-	for i, l := range dst {
+	
+	procID := ""
```

Comment:
Oh that explains it, It is left over code that i forgot to remove. Initially i tried to do what you suggested but it didn't work. Ok i'll have a second go at it,  @cbednarski once i fix this it should address your concerns too